src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -286,17 +286,22 @@ public function serialize() {
 		$data            = $this->fetch_data();
 		$data['post_id'] = $this->post_id;
 
-		return serialize( $data );
+		return wp_json_encode( $data );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
 	public function unserialize( $serialized ) {
-		$data          = unserialize( $serialized );
+		$data          = json_decode( $serialized, true );
+		array_walk( $data, static function ( &$data_entry ) {
+			if ( is_array( $data_entry ) ) {
+				$data_entry = (object)$data_entry;
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Expected 1 space(s) after cast statement; 0 found

<sub>hash: ff380f93b452ea5b7a98328d912a01b4</sub>